src/chat.ts

```diff
@@ -211,6 +211,14 @@ export class ChatProvider implements vscode.WebviewViewProvider {
         window.sourceryLS = {
           postMessage: vscode.postMessage,
         };
+
+        const updateTheme = () => {
+          const theme = document.body.classList.contains("vscode-light") ? "light" : "dark";
```

Comment:
Note that this class can be set to `vscode-light`, `vscode-dark` and `vscode-high-contrast`. 

High contrast is a dark theme so use `dark` for that and `vscode-dark`

Reply:
**suggestion:** make that a ocmment in the code here